### PR TITLE
Apply a consistent naming system to klasa

### PR DESCRIPTION
**Pull request by Tylertron1998**
### Description of the PR

Changed arguments to longer versions, msg => message, mes => response, start => runTime, and changed the commandLogging section to use the lengthened version; cmdLogging => commandLogging

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

- See description

### Semver Classification

- [x] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [x] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

File: guides/Advanced SettingGateway/SettingGatewayConfigurationUpdate.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ message.guild.configs.update('roles.administrator', '339943234405007361', messag
 // For retrocompatibility, the object overload is still available, however, this is much slower.
 // If you store objects literals in keys that do not take an array, this may break, prefer the
 // other overload or use nested SchemaPieces for full consistency.
-msg.guild.configs.update({ roles: { administrator: '339943234405007361' } }, msg.guild);
+msg.guild.configs.update({ roles: { administrator: '339943234405007361' } }, message.guild);
```
Comment (bdistin) [inline guides/Advanced SettingGateway/SettingGatewayConfigurationUpdate.md]: msg & message

File: src/lib/permissions/PermissionLevels.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ class PermissionLevels extends Collection {
 	 * Adds levels to the levels cache to be converted to valid permission structure
 	 * @since 0.2.1
 	 * @param {number} level The permission number for the level you are defining
-	 * @param {PermissionLevelOptions|symbol} obj Whether the level should break (stop processing higher levels, and inhibit a no permission error)
+	 * @param {PermissionLevelOptions|symbol} obj whether the level should break (stop processing higher levels, and inhibit a no permission error)
```
Comment (bdistin) [inline src/lib/permissions/PermissionLevels.js]: Whether should remain `Whether`

**Comment by MrJacz** [comment]
Could we get a changelog before this gets merged into master?

**Comment by Tylertron1998** [comment]
@bdistin any more requested changes on this?

**Comment by MrJacz** [comment]
Still requesting a changelog due to the fact this is `Semver: Major` and breaking even tho this is a development branch most users do use master so a changelog would be deeply appreciated.

**Pull request merged**